tests/conftest.py

```diff
@@ -151,6 +159,52 @@ def update_nested_obj(id_path, obj, i):
                     f.write(json.dumps(row, default=str) + "\n")
 
 
+# Debugging aids
```

Comment:
I threw these in here out of convenience, but could be convinced to put them into a test_utils if we feel strongly about that.

Reply:
This is fine!